### PR TITLE
docs: add docs for new cacheAcrossSpecs cy.session option

### PR DESCRIPTION
**Pull request by emilyrohrbough**

### CODE REVIEW

**Comment by emilyrohrbough** [comment]
TODO: so I don't forget -- update instrument panel images

File: content/api/commands/session.md [hunk 1]
```diff
@@ -135,14 +134,14 @@ The page and all active session data (cookies, `localStorage` and
...
+| ------------------ | ----------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `validate`         | `undefined` | Validates the newly-created or restored session.<br><br>Function to run immediately after the session is created and `setup` function runs or after a session is restored and the page is cleared. If this returns `false`, throws an exception, returns a Promise which resolves to `false` or rejects or contains any failing Cypress command, the session is considered invalid.<br><br>- If validation fails immediately after `setup`, the test will fail.<br>- If validation fails after restoring a session, `setup` will re-run. |
```
Comment (AtofStryker) [inline content/api/commands/session.md:139]: This section reads a bit weird to me. If `session` returns false, does an exception get thrown in addition to a promise being returned which resolves to `false` and/or rejects with a failing command?

"If this returns `false`, throws an exception, returns a Promise which resolves to `false` or rejects or contains any failing Cypress command"

Comment (emilyrohrbough) [reply content/api/commands/session.md:139]: These are examples of validate functions that will fail validation:
```js
validate = () => {
	expect(true).to.be.false // failed assertion throws (or something else that throws)
}

validate = () => {
	cy.get('#does_not_exist') // fails command
}

validate = () => {
	return false
}

validate = async () => {
	return Promise.resolve('false')
}

validate = async () => {
	return Promise.reject()
}
```

Comment (mjhenkes) [reply content/api/commands/session.md:139]: how about this? i slightly reordered the sentence. 

```suggestion
| `validate`         | `undefined` | Validates the newly-created or restored session.<br><br>Function to run immediately after the session is created and `setup` function runs or after a session is restored and the page is cleared. If this returns `false`, throws an exception, contains any failing Cypress command, or returns a Promise which rejects or resolves to `false`, the session is considered invalid.<br><br>- If validation fails immediately after `setup`, the test will fail.<br>- If validation fails after restoring a session, `setup` will re-run. |
```

**Review by mjhenkes — APPROVED** [review]

**Comment by emilyrohrbough** [comment]
@chrisbreiding I assume it'd be obvious you can't share state when on different machines.....but I can.

**Comment by emilyrohrbough** [comment]
@chrisbreiding good call
https://github.com/cypress-io/cypress/pull/23904#issuecomment-1258410002

File: content/api/commands/session.md [hunk 2]
```diff
@@ -422,6 +421,43 @@ cy.session('user', () => {
 })
 ```
 
+### Caching session data across specs
+
+If you want to use the same session across multiple specs, add
```
Comment (emilyrohrbough) [inline content/api/commands/session.md:426]:
```suggestion
If you want to use the same session across multiple specs in the same Cypress run, add
```

**Review by chrisbreiding — APPROVED** [review]

**Pull request merged**